sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/administration/models/AzureBlobFileListSource.java

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.ai.formrecognizer.documentanalysis.administration.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/** File list in Azure Blob Storage. */
+@Fluent
+public final class AzureBlobFileListSource {
+    /*
+     * Azure Blob Storage container URL.
+     */
+    @JsonProperty(value = "containerUrl", required = true)
+    private String containerUrl;
```

Comment:
Can the models in the `adminstraetion/models/` be publically viewed?

Reply:
yep they are public and available with the lib.

Reply:
Since `containerUrl`  and `fileList` are required, and this model class are autogenerated. We probably missing a autorest configuration option.
add below to autorest configuration
```
required-fields-as-ctor-args: true
```